docs/ibm-cloud-rules.md

```diff
@@ -871,6 +879,81 @@ paths:
 </tr>
 </table>
 
+<tr>
+<td><a href="#ibm-avoid-multiple-types">ibm-avoid-multiple-types</a></td>
+<td>warn</td>
+<td>Multiple types within a schema <code>type</code> field should be avoided due to potential impact on SDK code generation.
+</td>
+<td>oas3_1</td>
+</tr>
+
+
+### ibm-avoid-multiple-types
+<table>
+<tr>
+<td><b>Rule id:</b></td>
+<td><b>ibm-avoid-multiple-types</b></td>
+</tr>
+<tr>
+<td valign=top><b>Description:</b></td>
+<td>The OpenAPI 3.1 Specification allows multiple types to be used within a schema's <code>type</code> field to indicate that
+JSON instances of multiple types should validate cleanly against the schema definition.   This can be an effective tool when
+using the schema definition (i.e. API definition) to validate JSON instances.  However, when using the API definition to 
```

Comment:
> This can be an effective tool when using the schema definition (i.e. API definition) to validate JSON instances

It can be an effective tool for JSON validation but I'm not sure I would contextualize that within API definitions. IIRC, we'll be adding guidance to the API Handbook requiring users to use single types for schemas to prevent ambiguity from an API perspective and an SDK perspective, so we can probably capture that here.

Reply:
fixed